avocado/core/output.py

```diff
@@ -386,6 +381,8 @@ def reconfigure(args):
     if enabled:
         STD_OUTPUT.enable_outputs()
     else:
+        if getattr(args, "paginator", False) == "on" and TERM_SUPPORT.enabled:
```

Comment:
IIUC If show=none then it should not allow paging. However the checking here allows to enable paginator and show=none at same time. Am I wrong?

Reply:
You're right. I meant to put that on the other part of the conditional block.  Thanks for catching it!

Reply:
Tested with:

```diff
diff --git a/avocado/core/output.py b/avocado/core/output.py
index ac26850c..5027167f 100644
--- a/avocado/core/output.py
+++ b/avocado/core/output.py
@@ -376,10 +376,10 @@ def reconfigure(args):
     # TODO: Avocado relies on stdout/stderr on some places, re-log them here
     # for now. This should be removed once we replace them with logging.
     if enabled:
-        STD_OUTPUT.enable_outputs()
-    else:
         if getattr(args, "paginator", False) == "on" and TERM_SUPPORT.enabled:
             STD_OUTPUT.enable_paginator()
+        STD_OUTPUT.enable_outputs()
+    else:
         STD_OUTPUT.enable_stderr()
     STD_OUTPUT.print_records()
     if "app" in enabled:
```

And made sure paginator is never activated if ``-show=none` is given.